packages/nimble-components/src/accordion/specs/IxD.md

```diff
@@ -0,0 +1,193 @@
+
+| Component  | Content    | Organization |
+| ---------- | ---------- | ------------ |
+| Disclosure | Any        | Group labels |
+| Tree       | List items | Hierarchical |
+| ???        | List items | Group labels |
+
+[**QUESTION:** Maybe a new navigation-specific component?]
+
+## Design
+
+### Interactions & States
+
+Header buttons are used to expand and collapse associated panels of content. The client user can specify which panels (if any) are open by default.
+
+<p>
+<kbd><img src="spec-images/Accordion-1.png"></kbd>
+<br>
+<sub>Default (all panels closed)</sub>
+</p>
+
+Hovering over a header button indicates interactivity.
+
+<p>
+<kbd><img src="spec-images/Accordion-2.png"></kbd>
+<br>
+<sub>Hover</sub>
+</p>
+
+Clicking anywhere on a header button opens (if closed) or closes (if opened) the associated panel of content. [**QUESTION:** Do panels only display block content (one component per "line") or do they display flex content?]
+
+<p>
+<kbd><img src="spec-images/Accordion-3.png"></kbd>
+<br>
+<sub>Click</sub>
+</p>
+
+The client user can specify whether one panel or more than one panel can be opened at a time. If the former, clicking to open a closed panel will close the open panel (if applicable).
+
+<p>
+<kbd><img src="spec-images/Accordion-4.png"></kbd>
+<br>
+<sub>Click (one panel open at a time)</sub>
+</p>
+
+If a component in a panel is in an error state, the error state is indicated on the associated header button when closed (if not also when open).
+
+<p>
+<kbd><img src="spec-images/Accordion-5.png"></kbd>
+<br>
+<sub>Error</sub>
+</p>
+
+### Additional Options
+
+Client users may need/want to have additional buttons and/or icons in the header.
+
+<p>
+<kbd><img src="spec-images/Accordion-6.png"></kbd>
+<br>
+<sub>"Expand" button</sub>
+</p>
+
+### Nested Accordions
+
+Accordion panels can have one level of nested accordions. Nested accordions are always ghost style. [**QUESTION:** Can accordions be nested anywhere in the panel? Can there be multiple child accordions nested in an accordion?]
+
+<p>
+<kbd><img src="spec-images/Accordion-7.png"></kbd>
+<br>
+<sub>Nested accordion in "info" panel</sub>
+</p>
+
+### Keyboard Interactions
+
+| Key                | Description                                           |
+| ------------------ | ----------------------------------------------------- |
+| `TAB`              | Moves focus to next focusable element                 |
+| `SPACE` or `ENTER` | Opens/closes accordion section when header is focused |
+
+`TAB` to the accordion focuses the first header button (if not also visually indicating the entire accordion).
+
+<p>
+<kbd><img src="spec-images/Accordion-8.png"></kbd>
+<br>
+<sub>TAB to accordion</sub>
+</p>
+
+`TAB` focuses the next focusable element (if not also visually indicating the header button and/or the entire accordion). All focusable elements in the accordion are included in the `TAB` sequence.
```

Comment:
Should the tab sequence include focusable elements in collapsed sections? My inclination would be that it should only include focusable elements in open sections, but want to confirm.